scripts/gulp/tasks/gulpTsc.ts

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable no-console */
```

Comment:
Not that it matters too much, but, maybe a `.eslintrc` for the build directory?

Reply:
Yeah, was thinking about this but got lazy using the autosuggest. Will add